### PR TITLE
Add benchmark_video_blur.py

### PR DESCRIPTION
**Pull request by dev-bon**
This benchmarking script would measure the duration it takes to process videos at different sample rates and outputs the results on a text file.

### CODE REVIEW

**Comment by marcbelmont** [comment]
@adolfoarmas, could you please review this script, run it, and share the results?

**Comment by adolfoarmas** [comment]
These are the results I got, i ran various with sample 60 and one with 50:


| Sample | Output Size | Duration |
| ------ | ----------- | -------- |
|     60 |     2.25 MB |     14.7 |
|     60 |      2.0 MB |     21.4 |
|     60 |      2.0 MB |     17.1 |
|     60 |      2.0 MB |     18.4 |
|     50 |      2.0 MB |     19.3 |

Host Passmark [19912 ](https://www.cpubenchmark.net/cpu.php?cpu=AMD+Ryzen+5+5600G&id=4325)

**Test environment:** 
Ubuntu VM 
1 processor, 3 cores
RAM 8GB

**Comment by marcbelmont** [comment]
@dev-bon, do you need any further details?
@adolfoarmas, do you have any feedback on the script itself?

**Comment by dev-bon** [comment]
None on my end. The output satisfies the needed details on the metric sheet.

File: video-editor/benchmark_video_blur.py [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+import subprocess
+import time
+
+
+def parse_arguments():
+    parser = argparse.ArgumentParser(description="Benchmark Video Blur")
+    parser.add_argument(
+        "--video-editor-url", default="http://localhost:8081/process-video"
+    )
+    parser.add_argument("--sample", default="60,50")
```
Comment (adolfoarmas) [inline video-editor/benchmark_video_blur.py:19]: Is it possible to add a help text about this argument?
say: "One or more samples separated by a comma, for example: "60,50,30"

File: video-editor/benchmark_video_blur.py [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+        # Wait for the next poll_interval seconds before checking again
+        time.sleep(poll_interval)
+
+    # If we reach here, it means the API didn't become accessible within max_wait_time
+    return False
+
+
+def main():
+    args = parse_arguments()
+
```
Comment (adolfoarmas) [inline video-editor/benchmark_video_blur.py:141]: Adding a required arguments input validation before going forward may help?

Like: https://github.com/dev-bon/deep-license-plate-recognition/blob/289808ea07465c134825392a1b8d885beb955ae4/sftp_processor.py#L177

File: video-editor/benchmark_video_blur.py [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+
+def parse_arguments():
+    parser = argparse.ArgumentParser(description="Benchmark Video Blur")
+    parser.add_argument(
+        "--video-editor-url", default="http://localhost:8081/process-video"
+    )
+    parser.add_argument("--sample", default="60,50")
+    parser.add_argument("--video", default="assets/cars.mp4")
+    parser.add_argument("--blur-output", default="output/cars_blur.mp4")
```
Comment (adolfoarmas) [inline video-editor/benchmark_video_blur.py:21]: Is it possible to add a help text about this argument?
say: "Video file path where blur saves the already processed file"

File: video-editor/benchmark_video_blur.py [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+    file_exists = os.path.exists(file_path)
+
+    # Open the file in append mode if it exists, or create it if it doesn't
+    mode = "a" if file_exists else "w"
+
+    with open(file_path, mode) as file:
+        if not file_exists:
+            file.write(
+                "| {:<6} | {:<11} | {:>8} |\n".format(
+                    "Sample", "Output Size", "Duration"
```
Comment (adolfoarmas) [inline video-editor/benchmark_video_blur.py:39]: Nit: 

"Sample", "Output Size", "Processing Duration (s)"

or 

"Sample", "Output Size", "Processing Time (s)"

**Review by adolfoarmas — COMMENTED** [review]
Nice to have:

Video file duration in minutes included in results table, like:

|Video Duration (m)| Sample | Output Size | Process Duration (s) |
|---------------------| ------ | ----------- | -------- |
|NN.NN                    |     60 |     2.25 MB |     14.7 |
|NN.NN                    |     60 |      2.0 MB |     21.4 |
|NN.NN                    |     60 |      2.0 MB |     17.1 |
|NN.NN                    |     60 |      2.0 MB |     18.4 |
|NN.NN                    |     50 |      2.0 MB |     19.3 |

File: video-editor/benchmark_video_blur.py (hunk 1 above)
Comment (dev-bon) [reply video-editor/benchmark_video_blur.py]: Added on latest commit, also added video length on output

File: video-editor/benchmark_video_blur.py (hunk 2 above)
Comment (dev-bon) [reply video-editor/benchmark_video_blur.py]: Added required checks, available on latest commit

File: video-editor/benchmark_video_blur.py (hunk 3 above)
Comment (dev-bon) [reply video-editor/benchmark_video_blur.py]: Added on latest commit

File: video-editor/benchmark_video_blur.py (hunk 4 above)
Comment (dev-bon) [reply video-editor/benchmark_video_blur.py]: Added on latest commit

**Comment by marcbelmont** [comment]
@adolfoarmas, is there anything else? Please merge if it's ready.

**Comment by adolfoarmas** [comment]
| Video Duration (m) | Sample | Output Size | Processing Time (s) |
| ------------------ | ------ | ----------- | ------------------- |
|            0:02:41 |     30 |    178.5 MB |               585.5 |
|            0:02:41 |     40 |   178.75 MB |               519.8 |
|            0:02:41 |     50 |    178.5 MB |               469.6 |

results i got 
LGTM now @marcbelmont

**Review by adolfoarmas — APPROVED** [review]

**Pull request merged**